### ISSUE TITLE
Show all contributions by person (perhaps under profile tab?)

### ISSUE THREAD

**Issue by Peter9192** [research-software-directory/RSD-as-a-service#717]
I'm RSD-entry maintainer of a few projects and tools, but I'm contributor for many more. It would be nice if I could have an overview of all projects/tools for which I'm listed as a contributor.

**Comment by jmaassen** [comment]
Yes, indeed this would be nice to have. We are thinking about how to implement this. For contributions where your ORCID is know this is easy, but this is not always the case. 

We'll look into this in more detail.

**Comment by ewan-escience** [comment]
This was implemented in #1057 and released. See the [documentation](https://research-software-directory.org/documentation/users/user-settings/#public-profile) for more details.

**Comment by Peter9192** [comment]
Great that this is available now! 

In checking for completeness, I noticed that some entries are missing on my profile, and there seem to be different Peter Kalverla's in the system. For example, 

- The Peter Kalverla that's associated with [bartender](https://research-software-directory.org/software/bartender)
- The Peter Kalverla that's associated with [ewatercycle ii project](https://research-software-directory.org/projects/ewatercycle-ii),
- The Peter Kalverla that's associated with [ewatercycle software package](https://research-software-directory.org/software/ewatercycle),

Is there a way to find all projects/software that have my name listed as contributor, so I can make sure they link to the same profile?

**Comment by ewan-escience** [comment]
Under the hood, each software or project page has their own copies of contributors or team members. But when making profiles, we aggregate on ORCID, so the fact that e.g. your picture is different on those pages doesn't matter for the profile pages.

There isn't an easy way for you to see where you are listed, but we as admins have an extra tool where we can see that. In general, it's best to contact us if your ORCID is missing in many pages, or the maintainers of a page if its just a handful.

I can see your ORCID is missing on these 5 pages: [[1]](https://research-software-directory.org/software/setup-apptainer) [[2]](https://research-software-directory.org/software/i-vresse-workflow-builder) [[3]](https://research-software-directory.org/software/era5cli) [[4]](https://research-software-directory.org/software/storyboards) [[5]](https://research-software-directory.org/software/eucp-atlas). Shall I add your ORCID there?

**Comment by Peter9192** [comment]
Okay, I see! Thanks for the quick reply!

> I can see your ORCID is missing on these 5 pages: [[1]](https://research-software-directory.org/software/setup-apptainer) [[2]](https://research-software-directory.org/software/i-vresse-workflow-builder) [[3]](https://research-software-directory.org/software/era5cli) [[4]](https://research-software-directory.org/software/storyboards) [[5]](https://research-software-directory.org/software/eucp-atlas). Shall I add your ORCID there?

That'd be great, thanks! Does adding my ORCID there also add it to the Peter Kalverla's that are automatically found when you try to add a contributor to the software? That should help avoid this in the future.

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/17080502/11bcabd1-47d6-4504-ae4c-d14f90f1eff3)

**Comment by ewan-escience** [comment]
Done.

> Does adding my ORCID there also add it to the Peter Kalverla's that are automatically found when you try to add a contributor to the software?

Yes, it should.

**Comment by Peter9192** [comment]
Cool! I also realized that the "software I maintain" is different from the "software I contributed to". The latter is only visible on the public profile page. That seems fairly complete now.

One more question: can I edit my profile picture? Where does it even come from?

**Comment by ewan-escience** [comment]
> I also realized that the "software I maintain" is different from the "software I contributed to".

Yes, this is by design. Not every cocntributor needs to maintain the page (or even has an RSD account), and (more theoretically) pages can be maintained by someone who is not a contributor, think e.g. people who check the quality of a large amount of pages.

> can I edit my profile picture? Where does it even come from?

Currently not, but we might work on that. @dmijatovic implemented this, but I think the picture is the first one that the API returns, not having a well defined ordering.
